starter-code/src/app/components/my-home/my-home.component.html

```diff
@@ -0,0 +1,10 @@
+<p>
+  my-home works!
+</p>
+
+<ul>
+  <li *ngFor='let movie of movieArray'>
+   <p>{{movie.title}}</p>
+    <a [routerLink]="['/movie', movieSelected]">view more</a>
```

Comment:
What is `movieSelected`? 
You should have thee id of each movie with  `movie.id`